awscli/customizations/s3/s3handler.py

```diff
@@ -476,8 +475,8 @@ def _enqueue_multipart_upload_tasks(self, filename, payload=None):
         # First we need to create a CreateMultipartUpload task,
         # then create UploadTask objects for each of the parts.
         # And finally enqueue a CompleteMultipartUploadTask.
+        chunksize = enforce_chunksize_limits(self.chunksize)
 
-        chunksize = self.chunksize
         # Determine an appropriate chunksize if given an expected size.
         if self.params['expected_size']:
             chunksize = find_chunksize(int(self.params['expected_size']),
```

Comment:
Can this be consolidated into the line below with the check for expected size?  It was a little hard to follow given `find_chunksize` also calls `enforce_chunksize_limits`, but is only called when an expected size is given.